client/webserver/locales/parse_test.go

```diff
@@ -0,0 +1,213 @@
+      <label class="form-check-label" for="showPokes">
+        [[[Show pop-up notifications]]]
+      </label>
+    </div>
+    <div {{if not .UserInfo.Authed}} class="d-hide"{{end}}>
+    <span>Fiat Currency: </span><span id="fiatCurrency">{{.FiatCurrency}}</span>
+    </div>
+    <div id="fiatRateSources" {{if not .UserInfo.Authed}} class="d-hide"{{end}}>
+    <span class="mb-2" data-tooltip="[[[fiat_exchange_rate_msg]]]">
+     [[[fiat_exchange_rate_sources]]]:
+    <span class="ico-info"></span>
+    </span>
+    {{range $source, $enabled := .FiatRateSources}}
+    <div class="form-check">
+      <input class="form-check-input" type="checkbox" value="{{$source}}" id="{{$source}}" {{if $enabled}} checked {{end}}>
+      <label class="form-check-label" for="{{$source}}">{{$source}}</label>
+    </div>
+    {{end}}
+    </div>
+    <div>
+      <div id="exchanges" {{if eq (len .UserInfo.Exchanges) 0}} class="d-hide"{{end}}>
+        <h5>[[[registered dexes]]]</h5>
+        {{range $host, $xc := .UserInfo.Exchanges}}
+          <a href="/dexsettings/{{$host}}"><button class="bg2 selected"><div class=text-break>{{$host}}<span class="dex-settings-icon ico-settings"></span></div></button></a>
+        {{end}}
+      </div>
+      <br>
+      <div {{if not .UserInfo.Authed}} class="d-hide"{{end}}>
+        <p>
+        [[[simultaneous_servers_msg]]]
+        </p>
+        <button id="addADex" class="bg2 selected">[[[Add a DEX]]]</button>
+        <button id="importAccount" class="bg2 selected ms-2">[[[Import Account]]]</button>
+      </div>
+    </div>
+    <div>
+        <button id="changeAppPW" class="bg2 selected">[[[Change App Password]]]</button>
+    </div>
+    <div{{if not .UserInfo.Authed}} class="d-hide"{{end}}>
+      [[[seed_implore_msg]]]<br>
+      <button id="exportSeed" class="fs15 bg2 selected">[[[View Application Seed]]]</button>
+    </div>
+    <div>[[[Build ID]]]: <span id="commitHash" class="mono"></span></div>
+  </div>
+
+  {{- /* POP-UP FORMS */ -}}
+  <div id="forms" class="d-hide">
+    
+    {{- /* DEX ADDRESS */ -}}
+    <form class="position-relative text-start" id="dexAddrForm" autocomplete="off">
+      {{template "dexAddrForm" .}}
+    </form>
+
+    {{- /* REG ASSET SELECTOR */ -}}
+    <form class="d-hide" id="regAssetForm">
+      {{template "regAssetForm"}}
+    </form>
+    {{- /* CONFIRM REGISTRATION */ -}}
+    <form class="d-hide" id="confirmRegForm">
+      {{template "confirmRegistrationForm"}}
+    </form>
+
+    {{- /* AUTHORIZE IMPORT ACCOUNT */ -}}
+    <form class="d-hide" id="authorizeAccountImportForm">
+      {{template "authorizeAccountImportForm" .}}
+    </form>
+
+    {{- /* NEW WALLET */ -}}
+    <form class="d-hide" id="newWalletForm">
+      {{template "newWalletForm" }}
+    </form>
+
+    {{- /* CHANGE APP PASSWORD */ -}}
+    <form class="d-hide" id="changeAppPWForm">
+      {{template "changeAppPWForm"}}
+    </form>
```

Comment:
This entire stretch is what makes no sense to be in the test data IMO.  Sure it's "real world" but I don't see how it adds anything.  The pattern just repeats 6 times.

Reply:
Ok, trimmed it down